src/user-event/press/__tests__/press.real-timers.test.tsx

```diff
@@ -0,0 +1,304 @@
+
+    render(
+      <Text
+        onPress={logEvent('press')}
+        onPressIn={logEvent('pressIn')}
+        onPressOut={logEvent('pressOut')}
+      >
+        press me
+      </Text>
+    );
+    await userEvent.press(screen.getByText('press me'));
+
+    expect(getEventsName(events)).toEqual(['pressIn', 'press', 'pressOut']);
+  });
+
+  test('doesnt trigger on disabled Text', async () => {
+    const { events, logEvent } = createEventLogger();
+
+    render(
+      <Text
+        onPress={logEvent('press')}
+        onPressIn={logEvent('pressIn')}
+        onPressOut={logEvent('pressOut')}
+        disabled
+      >
+        press me
+      </Text>
+    );
+    await userEvent.press(screen.getByText('press me'));
+
+    expect(events).toEqual([]);
+  });
+
+  test('doesnt trigger on Text with disabled pointer events', async () => {
+    const { events, logEvent } = createEventLogger();
+
+    render(
+      <View pointerEvents="box-only">
+        <Text
+          onPress={logEvent('press')}
+          onPressIn={logEvent('pressIn')}
+          onPressOut={logEvent('pressOut')}
+        >
+          press me
+        </Text>
+      </View>
+    );
+    await userEvent.press(screen.getByText('press me'));
+
+    expect(events).toEqual([]);
+  });
+
+  test('works on TetInput', async () => {
+    const { events, logEvent } = createEventLogger();
+
+    render(
+      <TextInput
+        placeholder="email"
+        onPressIn={logEvent('pressIn')}
+        onPressOut={logEvent('pressOut')}
+      />
+    );
+    await userEvent.press(screen.getByPlaceholderText('email'));
+
+    expect(getEventsName(events)).toEqual(['pressIn', 'pressOut']);
+  });
+
+  test('does not call onPressIn and onPressOut on non editable TetInput', async () => {
+    const { events, logEvent } = createEventLogger();
+
+    render(
+      <TextInput
+        placeholder="email"
+        editable={false}
+        onPressIn={logEvent('pressIn')}
+        onPressOut={logEvent('pressOut')}
+      />
+    );
+    await userEvent.press(screen.getByPlaceholderText('email'));
+    expect(events).toEqual([]);
+  });
+
+  test('does not call onPressIn and onPressOut on TetInput with pointer events disabled', async () => {
+    const { events, logEvent } = createEventLogger();
+
+    render(
+      <TextInput
+        placeholder="email"
+        pointerEvents="box-none"
+        onPressIn={logEvent('pressIn')}
+        onPressOut={logEvent('pressOut')}
+      />
+    );
+    await userEvent.press(screen.getByPlaceholderText('email'));
+    expect(events).toEqual([]);
+  });
+
+  test('press is accessible directly in userEvent', async () => {
+    const mockOnPress = jest.fn();
+
+    render(
+      <Pressable onPress={mockOnPress}>
+        <Text>press me</Text>
+      </Pressable>
+    );
+
+    await userEvent.press(screen.getByText('press me'));
+
+    expect(mockOnPress).toHaveBeenCalled();
+  });
+
+  test('warns about using real timers with userEvent', async () => {
```

Comment:
[discussion, tradeoffs] I think we should allow the regular `press` to be called without warning for real timers, this will decrease slightly how realistic this simulation is but it will ease adoption by the users since it will not require the to switch to fake timers, which migth be a hurdle for some of the users.

Perhaps we could enable more realistic mode for advanced users using some `userEvent` option or by detecting fake timers?

@pierrezimmermannbam wdyt?

Reply:
If this is resulting from`Pressability` implementation then we should accept this 130 ms delay as a fact. Such delay for one test is insignificant but for numerous tests will quickly add up to meaningful numbers. 

Perhaps what we could do is to issue the warning only in `Pressability` branch.

Reply:
Yes it's a possibility but I fear it would make things more confusing for users and I don't see anyone that's not willing to use the API because they're not using fake timers use it only for Text and TextInputs, especially since using it on those will be much less common